site/docs/spec.md

```diff
@@ -333,6 +333,25 @@ Table metadata is stored as JSON. Each table metadata change creates a new table
 
 The atomic operation used to commit metadata depends on how tables are tracked and is not standardized by this spec. See the sections below for examples.
 
+### Delete Format
+
+This section details how to encode row-level deletes in Iceberg metadata. Row-level deletes are not supported in the current format version 1. This part of the spec is not yet complete and will be completed as format version 2.
+
+#### Position-based Delete Files
+
+Position-based delete files identify rows in one or more data files that have been deleted. It has the schema named "position_based_delete_file" with a struct of the following fields:
```

Comment:
Let's rephrase the second sentence so that it matches the ones above:

> Position-based delete files store `file_position_delete`, a struct with the following fields: